Tasks/ServiceFabricDeploy/deploy.ps1

```diff
@@ -31,6 +31,7 @@ try {
     $compressPackage = [System.Boolean]::Parse((Get-VstsInput -Name compressPackage))
     $skipUpgrade =  [System.Boolean]::Parse((Get-VstsInput -Name skipUpgradeSameTypeAndVersion))
     $skipValidation =  [System.Boolean]::Parse((Get-VstsInput -Name skipPackageValidation))
+    $unregisterUnusedVersions = [System.Boolean]::Parse((Get-VstsInput -Name unregisterUnusedVersions))
```

Comment:
Did you run all the set-up steps in [contribute](https://github.com/Microsoft/vsts-tasks/blob/master/docs/contribute.md), in particular: [make-dependencies](https://github.com/Microsoft/vsts-tasks/blob/master/docs/contribute.md#make-dependencies)?